fjord/feedback/jinja2/feedback/android_feedback.html

```diff
@@ -0,0 +1,192 @@
+                      {% csrf_token %}
+
+                      <div class="input-box">
+                        <input type="submit" value="{{ _('Send Feedback') }}" id="form-submit-btn" class="complete btn submit" disabled="disabled">
+                        </input>
+                      </div><!-- .input-box -->
+
+                      <div class="privacy">
+                        {% trans %}
+                          For your privacy, please do not include any personal information
+                          in your feedback.
+                        {% endtrans %}
+                      </div><!-- .privacy -->
+                    </form>
+                  </div><!-- .ask -->
+
+                </div><!-- .sad -->
+              </section>
+            </div><!-- #moreinfo.card -->
+          </div><!-- .deck -->
+        </div><!-- .content -->
+
+    {% endblock %}
+  </div>
+  <div class="footer happy">
+    <a id="maybe-later" data-event="FeedbackMaybeLater" href="#">
+      {% trans %}
+        Maybe Later
+      {% endtrans %}
+    </a>
+    <a id="no-thanks" data-event="FeedbackClose" href="#">
+      {% trans %}
+        No Thanks
+      {% endtrans %}
+    </a>
+  </div><!-- .footer.happy -->
+  <div class="footer sad">
+    <div class="help-section">
+      <svg viewBox="0 0 156 156">
+        <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+          <g transform="translate(3.000000, 2.000000)">
+              <ellipse stroke="#DE1920" stroke-width="6" fill="#FFFFFF" transform="translate(75.008751, 75.991249) rotate(-270.000000) translate(-75.008751, -75.991249) " cx="75.008751" cy="75.991249" rx="75.008751" ry="75.008751"></ellipse>
+              <path d="M114.879498,75.7124061 L150.016995,75.7124061 C149.8667,34.4144668 116.341953,0.982498052 75.008751,0.982498052 C74.7295491,0.982498052 74.4507036,0.98402351 74.1722222,0.987066548 L74.1722222,36.1205023 C74.357816,36.1179759 74.543714,36.1167085 74.729908,36.1167085 C96.7198153,36.1167085 114.581115,53.7933778 114.879498,75.7124061 Z M74.729908,116.423475 L74.729908,150.999529 C33.4319688,150.849198 0,117.324451 0,75.991249 C0,75.8982618 0.000169204083,75.805314 0.000507321153,75.7124061 L34.5803184,75.7124061 C34.5777919,75.8979999 34.5765246,76.083898 34.5765246,76.270092 C34.5765246,98.4461933 52.5538067,116.423475 74.729908,116.423475 Z" fill="#DE1920" transform="translate(75.008497, 75.990995) rotate(-270.000000) translate(-75.008497, -75.990995) "></path>
+              <ellipse stroke="#DE1920" stroke-width="6" fill-opacity="0" transform="translate(74.729908, 76.270092) rotate(-270.000000) translate(-74.729908, -76.270092) " cx="74.729908" cy="76.270092" rx="40.1533834" ry="40.1533834"></ellipse>
+          </g>
+        </g>
+      </svg>
+      {% trans %}
+        Visit our <a href="https://support.mozilla.org/en-US/products/mobile/popular-articles-android?as=u&utm_source=inproduct">Support Site</a> to get answers for common issues.
```

Comment:
You shouldn't hard-code the locale here. This SUMO link should be:

https://support.mozilla.org/products/mobile/popular-articles-android?as=u&utm_source=inproduct